src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -172,7 +172,13 @@ public function section($key, $template=null)
 
 		if ($template === null)
 		{
-			$template = 'block_section';
```

Comment:
Usually every position should have at least `block_section`. But I agree, to be save we shouldn't remove it.